### PR TITLE
header UI

### PR DESCRIPTION
**Pull request by jstsanam**
Problem statement: To make Header for the website

Preview of the header:
image

### CODE REVIEW

**Comment by som-matrix** [comment]
There are certain rules you need to follow @jstsanam while opening PR:
1. Add PR description , like what is the problem you are solving and what changes are you making in the PR.
2. Add screenshots in the PR if there is no preview link setup

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <button className="p-2 rounded-full" style={{ backgroundColor: "#1f1f1f" }}>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            height="24px"
+            viewBox="0 -960 960 960"
+            width="24px"
+            fill="#e8eaed"
+          >
+            <path d="M160-120v-480l320-240 320 240v480H560v-280H400v280H160Z" />
+          </svg>
```
Comment (som-matrix) [inline src/components/Header.tsx:15]: for later , we can move this to icons directory in components folder

**Review by som-matrix — COMMENTED** [review]
Requested changes , other changes looks good

File: src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+export default function Header() {
+  return (
+    <header className="flex justify-between items-center m-2">
+      <img src="header-logo.png" alt="logo" className="h-6 w-6" />
+      <div className="flex items-center gap-2">
+        <button className="p-2 rounded-full" style={{ backgroundColor: "#1f1f1f" }}>
```
Comment (som-matrix) [inline src/components/Header.tsx:6]: no inline styles use tailwindcss please

File: src/components/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            >
+              <path d="M784-120 532-372q-30 24-69 38t-83 14q-109 0-184.5-75.5T120-580q0-109 75.5-184.5T380-840q109 0 184.5 75.5T640-580q0 44-14 83t-38 69l252 252-56 56ZM380-400q75 0 127.5-52.5T560-580q0-75-52.5-127.5T380-760q-75 0-127.5 52.5T200-580q0 75 52.5 127.5T380-400Z" />
+            </svg>
+          </div>
+          <input
+            type="search"
+            id="default-search"
+            className="p-2.5 ps-11 text-sm rounded-full"
+            placeholder="What do you want to play?"
+            style={{ backgroundColor: "#1f1f1f", width: "25rem" }}
```
Comment (som-matrix) [inline src/components/Header.tsx:34]: again here , no inline styles use tailwind css

File: src/components/Header.tsx (hunk 1 above)
Comment (jstsanam) [reply src/components/Header.tsx:15]: Okay sure we can.

File: src/components/Header.tsx (hunk 2 above)
Comment (jstsanam) [reply src/components/Header.tsx:6]: Okay

**Pull request merged**